src/main/scala/temple/Main.scala

```diff
@@ -13,7 +15,7 @@ object Main extends App {
   try {
     val config = new TempleConfig(args)
     config.subcommand match {
-      case Some(config.Generate) => Application.generate(config)
+      case Some(config.Generate) => Application.generate(config, PoliceSergeantNicholasAngel)
```

Comment:
Wtf is this and why is it the default behaviour of our program?

Reply:
Mornin angle